### PR TITLE
Fix scrolling styles and others

### PR DESCRIPTION
**Pull request by kienan**
**What & Why:**
*Finally* fixing the styles that caused scrolling to be super choppy. The main containers were fixed to the window size, so content had to be re-rendered on scroll. Now it should all be rendered on load and scrolling is significantly smoother. 

**Details:**
- The body's background color is no longer blue -- that was a weird attribute to deal with.
- The main container is no longer fixed
- The footer now stays at the bottom of the content on the welcome screens and in the sidebar
- The sidebar now scrolls correctly when the screen height is reduced
- Minor style changes to the about page to add more padding

Later on I'll go back through styles and start to do more refactoring to make sure we're following best practices with naming, etc.

### CODE REVIEW

**Comment by atallahhezbor** [comment]
Good stuff. Much needed. 
Did you want additional refactoring in this PR or another one?

File: app/assets/stylesheets/about.scss [hunk 1]
```diff
@@ -1,201 +1,206 @@
...
-	box-shadow: 0px 1px 2px 0px #B8B8B8;
-	padding-right: 5px;
-	margin-bottom: 5px;
-	margin-left: 0px;
-	margin-right: 0px;
-	margin-top:5px;
-	padding-top: 10px;	
+.about-container, .us-container {
+
+}
```
Comment (atallahhezbor) [inline app/assets/stylesheets/about.scss]: worth having empty rules?

File: app/assets/stylesheets/sidebar.scss [hunk 2]
```diff
@@ -6,12 +6,15 @@
     }
 }
 
-#nav-footer {
-  left: auto;
-  right: auto;
+.sidebar__links {
+  
+}
```
Comment (atallahhezbor) [inline app/assets/stylesheets/sidebar.scss]: Same as above, good to delete this?

**Comment by kienan** [comment]
@atallahhezbor fixed up the empty rules -- good catch. 

There are so many things we need to refactor that it's definitely going to have to come later. It might be a good exercise for all of us on the team to refactor one file or something like that.

**Comment by atallahhezbor** [comment]
Looks good then! Merging

**Pull request merged**